core/src/it/scala/com/github/mjakubowski84/parquet4s/ParquetStreamWriterSpec.scala

```diff
@@ -0,0 +1,40 @@
+package com.github.mjakubowski84.parquet4s
+
+import java.io.{ByteArrayOutputStream, FileOutputStream}
+
+import org.scalatest.{BeforeAndAfter, FreeSpec, Matchers}
+
+case class Test(
+                 t: String,
+                 f: Option[Int]
+               )
+class ParquetStreamWriterSpec extends FreeSpec
+with Matchers
+with BeforeAndAfter with SparkHelper {
+  import sparkSession.implicits._
+
+
+  "write parquet to outputstream" - {
+    val data = Seq(Test("a", None), Test("b", Some(1)))
+    val outputStream = new ByteArrayOutputStream()
+    val outfile = new StreamOutputFile(outputStream)
+    import ParquetWriter._
+    ParquetWriter.writeS(outfile,data)
+    outputStream.close()
+    println(outputStream)
+    outputStream.size() > 1 should equal(true)
```

Comment:
improve test by 
1. creating seq of generated content
2. writing generated content to parquet file using existing writer
3. writing generated content to parquet stream using new writer 
4. compare content is the same (turn stream into file, read file into stream, etc)